apps/deploy-web/src/lib/nextjs/proxyRequest/proxyRequest.ts

```
@@ -0,0 +1,107 @@
+
+  Object.entries(req.headers).forEach(([key, value]) => {
+    if (headers.has(key)) return;
+
+    const lowKey = key.toLowerCase();
+    if (lowKey === "host" || lowKey === "connection" || lowKey === "cookie") return;
+    if (HEADERS_TO_SKIP.has(lowKey)) return;
+    if (!Array.isArray(value)) {
+      headers.set(key, value as string);
+      return;
+    }
+    value.forEach(v => headers.append(key, v));
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)